### ISSUE TITLE
Compatibility issue for version 0.19.8 GLIBC_2.29' not found

### ISSUE THREAD

**Issue by rchancey** [amida-tech/blue-button#260]
I am getting this when using this with node 14.7.0. And the latest blue-button. Seems its from https://github.com/libxmljs/libxmljs. 
Any suggestions on how to get past this? 

Also runs great on my local Mac but when pushing to AWS lambda it fails.

**Comment by Manhydra** [comment]
@rchancey Yes, many are experiencing this issue with libxmljs@0.19.10. A work-around would be to delete the current `node_modules` directory from your project then running `npm install --build-from-source`

**Comment by rchancey** [comment]
@Manhydra 

trying..getting this: services:1.22.0 - :bootstrap.js:55  Detected an unhandled exception Error: Could not locate the bindings file. Tried:
 → /opt/atlassian/pipelines/agent/build/node_modules/@amida-tech/blue-button-xml/node_modules/libxmljs/build/xmljs.node
 → /opt/atlassian/pipelines/agent/build/node_modules/@amida-tech/blue-button-xml/node_modules/libxmljs/build/Debug/xmljs.node

I installed the ilbxmljs early on and from source.. but I think the blue button is installing its own version internal somehow?

**Comment by rchancey** [comment]
For those who are experiencing the same , we ended up having to build on the same image as AWS Lambda. We also had to fork the lib and back off a version since the current version did not run on windows. Since its native it has to all line up. WE hit every issue in this thread.. was a miserable 2 days. This parser is GEAT but its also HUGE and pushed our image size over the top. We ended up having to deploy this as a stand a lone service :-( .. but we did get it working and its ok.. its also slow :-( like > 4 seconds to parse a doc).. some work is done to make this dude faster. But for those who are using this with lambda, use the AWS image was key for us.  
image: amazonlinux:2018.03.0.20220609.0-with-sources

**Comment by joshmc82** [comment]
@rchancey can you share the AMI id? I don't see the image attached. We have the XML to JSON functionality working in Lambda but can't get JSON to XML due to these issues. I can't even get it to install on my Mac. From our experience, CCD to JSON performance is reasonable if you give it enough RAM.
